libraries/chain/steem_evaluator.cpp

```diff
@@ -580,6 +584,70 @@ void account_update_evaluator::do_apply( const account_update_operation& o )
 
 }
 
+void account_update2_evaluator::do_apply( const account_update2_operation& o )
+{
```

Comment:
The beginning of the evaluator needs a hardfork check so the new op is not accepted prior to the hardfork time.